syntax = "proto3";

message TicketRequest {
    string user_id = 1;
    int32 no_of_tickets = 2;
}

message TicketCreate {
    string board = 1;
}

message TicketResponse {
    repeated TicketCreate ticketgenerated = 1;
}

service Tickets {
    rpc Ticket (TicketRequest) returns (TicketResponse);
}
